tasks/components/layout/DashboardDisplay.tsx

```diff
@@ -43,10 +44,12 @@ export type DashboardDisplayProps = {
 export const DashboardDisplay = ({
   language,
   wards,
-  organizations
+  organizations,
+  width
 }: PropsWithLanguage<DashboardDisplayTranslation, DashboardDisplayProps>) => {
   const translation = useTranslation(language, defaultDashboardDisplayTranslations)
   const router = useRouter()
+  const columns = width === undefined ? 3 : Math.max(Math.floor(width / 260), 1)
```

Comment:
width could be `null`
```suggestion
  const columns = !!width ? 3 : Math.max(Math.floor(width / 260), 1)
```

Reply:
magic number `260` where does this come from?
Add a comment explaining this number